actions/push-to-gar-docker/action.yaml

```diff
@@ -89,37 +95,17 @@ runs:
           exit 1
         fi
         echo "project=${PROJECT}" >> ${GITHUB_OUTPUT}
-    - name: Get repository name
-      id: get-repository-name
-      shell: bash
-      run: |
-        REPO_NAME=$(echo ${{ github.repository }} | awk -F'/' '{print $2}')
-        echo "repo_name=${REPO_NAME}" >> ${GITHUB_OUTPUT}
+    - name: Login to GAR
+      uses: grafana/shared-workflows/actions/login-to-gar@main
```

Comment:
will change this from main to pinned in a followup PR which will resolve https://github.com/grafana/shared-workflows/issues/96

Reply:
But when is that pinning then updated? We don't yet have any real versions here.

Reply:
yes I was just talking about this to Iain - I guess we should start versioning this thing soon so we can use dependabot for that.

Reply:
Agreed 🙂 Until we have that, though, we should refrain from internal pinning IMO.

Reply:
https://github.com/grafana/shared-workflows/issues/68